packages/terra-clinical-result/src/flowsheetresultcell/FlowsheetResultCell.jsx

```diff
@@ -0,0 +1,277 @@
+   */
+  paddingStyle: PropTypes.oneOf(['none', 'standard', 'compact']),
+  /**
+   * Override that shows an Error display. Used when there is a known error or problem when retrieving or assembling the clinical result data.
+   */
+  hasResultError: PropTypes.bool,
+  /**
+   * Override that shows a known "No Data" display. Used when there is known to be no value for a given clinical result concept at a specific datetime.
+   */
+  hasResultNoData: PropTypes.bool,
+  /**
+   * @private
+   * The intl object to be injected for translations.
+   */
+  intl: intlShape.isRequired,
+};
+
+const defaultProps = {
+  resultDataSet: [],
+  hideUnit: false,
+  paddingStyle: 'compact',
+  hasResultError: false,
+  hasResultNoData: false,
+};
+
+const FlowsheetResultCell = (props) => {
+  const {
+    resultDataSet,
+    hideUnit,
+    paddingStyle,
+    hasResultError,
+    hasResultNoData,
+    intl,
+    ...customProps
+  } = props;
+  const containerDiv = useRef(null);
+  const [contentWidth, setContentWidth] = useState(null);
+  const [numericOverflow, setNumericOverflow] = useState(false);
+
+  useEffect(() => {
+    if (containerDiv.current && resultDataSet[0]) {
+      if (checkIsOfType(resultDataSet[0], 'NUMERIC')) {
+        if (!contentWidth) {
+          setContentWidth(containerDiv.current.children[0].getBoundingClientRect().width);
+        }
+        if (containerDiv.current.getBoundingClientRect().width <= contentWidth && !numericOverflow) {
+          setNumericOverflow(true);
+        } else if (containerDiv.current.getBoundingClientRect().width > contentWidth) {
+          setNumericOverflow(false);
+        }
+      }
+    }
+  }, [resultDataSet, contentWidth, numericOverflow]);
+
+  let flowsheetResultCellDisplay = null;
+
+  if (hasResultError) {
+    flowsheetResultCellDisplay = <div className={cx(['primary-display', 'error'])}><ResultError /></div>;
+  } else if (hasResultNoData) {
+    flowsheetResultCellDisplay = <div className={cx('primary-display')}><NoData /></div>;
+  } else {
+    const createResultsDisplay = (resultSet) => {
+      let resultKeyID;
+      let resultsDisplay = [];
+      let resultsInnerDisplay;
+      let additionalResultCount = 0;
+      let additionalResultInnerDisplay;
+      const additionalResultInterpretations = [];
+      let additionalResultInterpretationIndicator;
+      let endAccessoryIcons;
+      let primaryResultHasComment = false;
+      let primaryResultIsModified = false;
+      let primaryResultIsUnverified = false;
+      let primaryDisplayWithInterpretation = false;
+
+      if (!resultSet || !resultSet.length) {
+        resultKeyID = 'Error';
+        resultsDisplay.push(<div key={(`ClinicalResultDisplay-${resultKeyID}`)} className={cx(['primary-display', 'error'])}><ResultError /></div>);
+      } else {
+        resultsDisplay = [];
+        additionalResultCount = resultSet.length - 1;
+        for (let i = 0; i < resultSet.length; i += 1) {
```

Comment:
Convert this to resultSet.forEach((result, i) => { ... } );

Reply:
added forEach() here: https://github.com/cerner/terra-clinical/pull/574/commits/4015e0f2acf433faa34429946f05e3db8a2ebebc#diff-c342b09c8f0ec780dc2d3e76342313b9R286